AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-	package="de.duenndns.ssl"
 	android:versionCode="1"
```

Comment:
Moving the package declaration from the manifest into the gradle build description should be a atomic commit. Consider adding this change to https://github.com/ge0rg/MemorizingTrustManager/pull/71/commits/d98391c42f3d9275ba675a887fedba4ba9681493